src/gradient.rs

```diff
@@ -411,7 +411,10 @@ impl<const N: usize, CS: ColorSpace> ColorStopsSource for [OpaqueColor<CS>; N] {
 
 #[cfg(test)]
 mod tests {
+    extern crate alloc;
+    extern crate std;
```

Comment:
This is neat! I didn't realise that you could have more than one `extern crate x` in a single crate.